src/devcenter/azext_devcenter/manual/custom.py

```diff
@@ -480,17 +480,19 @@ def devcenter_environment_update(
     cmd,
     dev_center,
     project_name,
-    user_id,
     environment_name,
     description=None,
     catalog_name=None,
     catalog_item_name=None,
     parameters=None,
     scheduled_tasks=None,
     tags=None,
+    user_id=None
```

Comment:
I think you may need to add "Default: me" for the help in params.py for this command since we set this to None and not me in the arguments.